multigeiger/transmission.h

```diff
@@ -5,6 +5,14 @@
 #ifndef _TRANSMISSION_H_
 #define _TRANSMISSION_H_
 
+// Sensor-PINS.
+// They are called PIN, because in the first days of Feinstaub sensor they were
+// really the CPU-Pins. Now they are 'virtual' pins to distinguish different sensors.
+// Since we send to sensor.community, we have to use their numbers.
+#define XPIN_NO_XPIN 0
```

Comment:
so 0 does not exist as a PIN?

Reply:
Yes because it was a hardware pin number and they always start from 1